Makefile

```diff
@@ -1,18 +1,34 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
```

Comment:
One of the use cases for this is people building our images and pushing to their own repos.  Then could set their overrides in a single place and even commit in version control without having to set ENV vars all the time.  I personally hate ENV vars for lack of reproducibility.